snmp.yml

```diff
@@ -1565,3 +1565,946 @@ fortinet_fortigate:
+    - name: mtxrHlFiveVoltage
+      oid: 1.3.6.1.4.1.14988.1.1.3.3
+      indexes:
+        - labelname: mtxrSerialNumber
+          type: OctetString
+      lookups:
+        - labels: [mtxrSerialNumber]
+          labelname: mtxrSerialNumber
+          oid: 1.3.6.1.4.1.14988.1.1.7.3
+        - labels: [mtxrSerialNumber]
+          labelname: mtxrFirmwareVersion
+          oid: 1.3.6.1.4.1.14988.1.1.7.4
+    - name: mtxrHlTwelveVoltage
+      oid: 1.3.6.1.4.1.14988.1.1.3.4
+      indexes:
+        - labelname: mtxrSerialNumber
+          type: OctetString
+      lookups:
+        - labels: [mtxrSerialNumber]
+          labelname: mtxrSerialNumber
+          oid: 1.3.6.1.4.1.14988.1.1.7.3
+        - labels: [mtxrSerialNumber]
+          labelname: mtxrFirmwareVersion
+          oid: 1.3.6.1.4.1.14988.1.1.7.4
+    - name: mtxrHlSensorTemperature
+      oid: 1.3.6.1.4.1.14988.1.1.3.5
+      indexes:
+        - labelname: mtxrSerialNumber
+          type: OctetString
+      lookups:
+        - labels: [mtxrSerialNumber]
+          labelname: mtxrSerialNumber
+          oid: 1.3.6.1.4.1.14988.1.1.7.3
+        - labels: [mtxrSerialNumber]
+          labelname: mtxrFirmwareVersion
+          oid: 1.3.6.1.4.1.14988.1.1.7.4
+    - name: mtxrHlCpuTemperature
+      oid: 1.3.6.1.4.1.14988.1.1.3.6
+      indexes:
+        - labelname: mtxrSerialNumber
+          type: OctetString
+      lookups:
+        - labels: [mtxrSerialNumber]
+          labelname: mtxrSerialNumber
+          oid: 1.3.6.1.4.1.14988.1.1.7.3
+        - labels: [mtxrSerialNumber]
+          labelname: mtxrFirmwareVersion
+          oid: 1.3.6.1.4.1.14988.1.1.7.4
+    - name: mtxrHlBoardTemperature
+      oid: 1.3.6.1.4.1.14988.1.1.3.7
+      indexes:
+        - labelname: mtxrSerialNumber
+          type: OctetString
+      lookups:
+        - labels: [mtxrSerialNumber]
+          labelname: mtxrSerialNumber
+          oid: 1.3.6.1.4.1.14988.1.1.7.3
+        - labels: [mtxrSerialNumber]
+          labelname: mtxrFirmwareVersion
+          oid: 1.3.6.1.4.1.14988.1.1.7.4
+    - name: mtxrHlVoltage
+      oid: 1.3.6.1.4.1.14988.1.1.3.8
+      indexes:
+        - labelname: mtxrSerialNumber
+          type: OctetString
+      lookups:
+        - labels: [mtxrSerialNumber]
+          labelname: mtxrSerialNumber
+          oid: 1.3.6.1.4.1.14988.1.1.7.3
+        - labels: [mtxrSerialNumber]
+          labelname: mtxrFirmwareVersion
+          oid: 1.3.6.1.4.1.14988.1.1.7.4
+    - name: mtxrHlActiveFan
+      oid: 1.3.6.1.4.1.14988.1.1.3.9
+      indexes:
+        - labelname: mtxrSerialNumber
+          type: OctetString
+      lookups:
+        - labels: [mtxrSerialNumber]
+          labelname: mtxrSerialNumber
+          oid: 1.3.6.1.4.1.14988.1.1.7.3
+        - labels: [mtxrSerialNumber]
+          labelname: mtxrFirmwareVersion
+          oid: 1.3.6.1.4.1.14988.1.1.7.4
+    - name: mtxrLicUpgrUntil
+      oid: 1.3.6.1.4.1.14988.1.1.4.2
+      indexes:
```

Comment:
Can you elaborate on why this means it should be removed? It is being reported appropriately in my testing.